src/css-loader-core/loader.js

```diff
@@ -1,112 +1,32 @@
-// Copied from https://github.com/css-modules/css-modules-loader-core
```

Comment:
I think we can move the loader and parser files to the `src` directory directly. The directory was just a copy-and-paste from the library as a hot fix and unnecessary now.